internal/provider/resource_ssh_key.go

```
@@ -0,0 +1,49 @@
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+)
+
+func NewSshKeyResource() resource.Resource {
+	return &SshKeyResource{}
+}
+
+type SshKeyResource struct {
+	client *ProviderConfig
```

golangci-lint (1.64.8)

field `client` is unused

(unused)